### PR TITLE
GP2-1363: Drop custom Django view serving /markets/

### PR DESCRIPTION
**Pull request by stevejalim**
This changeset removes the Django view serving /markets/ because this URL path is required for pages that will be ported from Great V1.

The view is not being moved to respond to a different route - it is no longer required, so it is safe to remove.

Related tests were removed.

Other tests that used /markets/ as an example URL have been updated so that they still pass

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/TICKET_ID_HERE
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/urls.py [hunk 1]
```diff
@@ -20,7 +20,6 @@ def anonymous_user_required(function):
 
 
 urlpatterns = [
-    path('markets/', views.MarketsView.as_view(), name='markets'),
```
Comment (webbyfox) [inline core/urls.py:23]: dont we need to get rid of this MarketView as well?

Comment (stevejalim) [reply core/urls.py:23]: Yep - janky keyboard here meant the edits on views.py didn't get saved. Updating now

File: core/views.py [hunk 2]
```diff
@@ -88,31 +88,6 @@ def __init__(self):
...
-class MarketsView(GA360Mixin, TemplateView):
-    def __init__(self):
-        super().__init__()
-        self.set_ga360_payload(
-            page_id='Markets',
-            business_unit='MarketsUnit',
-            site_section='markets',
-        )
-
-    template_name = 'core/markets.html'
```
Comment (webbyfox) [inline core/views.py:100]: Does this template need to remove as well, then?

Comment (stevejalim) [reply core/views.py:100]: Yep - thanks! 🤦

**Review by webbyfox — APPROVED** [review]
LGTM

**Pull request merged**